src/main/java/uk/gov/companieshouse/overseasentitiesapi/validation/DueDiligenceValidator.java

```diff
@@ -85,7 +85,8 @@ private boolean validateAmlNumber(String amlNumber, Errors errors, String loggin
         String qualifiedFieldName = getQualifiedFieldName(
                 OverseasEntitySubmissionDto.DUE_DILIGENCE_FIELD,
                 DueDiligenceDto.AML_NUMBER_FIELD);
-        return StringValidators.isLessThanOrEqualToMaxLength(amlNumber, 256, qualifiedFieldName, errors, loggingContext);
+        return StringValidators.isNotBlank(amlNumber, qualifiedFieldName, errors, loggingContext)
```

Comment:
I think `AML_NUMBER_FIELD` is optional. We should just check the length of the field when is not null, this way you make it required. If it has become required we need to change web and spec

Reply:
We have to fix this bug, the details are here:

https://companieshouse.atlassian.net/browse/ROE-1510

Reply:
We can do that, but the expected result on the ticket will need to be changed.

Reply:
Updated the fix and the the ticket.